chapters/memory.adoc

```diff
@@ -1709,6 +1720,15 @@ ifdef::VK_EXT_metal_objects[]
     pname:exportObjectType member must: be
     ename:VK_EXPORT_METAL_OBJECT_TYPE_METAL_BUFFER_BIT_EXT
 endif::VK_EXT_metal_objects[]
+ifdef::VK_EXT_external_memory_metal[]
+  * If the parameters define an import operation and the external handle is
+    a ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLTEXTURE_BIT_EXT, then
+    pname:pNext must: include a slink:VkMemoryDedicatedAllocateInfo.
+  * If the parameters define an import operation and the external handle is
+    a ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLTEXTURE_BIT_EXT, then
```

Comment:
This should be the VU for `VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLHEAP_BIT_EXT`, but now that I take a second look at it, I don't see why it can't be merged to the one above

Reply:
Merged into above VU